tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1977,9 +1982,9 @@ def adjust_hue(image, delta, name=None):
 
   Usage Example:
     ```python
-    >> import tensorflow as tf
-    >> x = tf.random.normal(shape=(256, 256, 3))
-    >> tf.image.adjust_hue(x, 0.2)
+    >>> import tensorflow as tf
```

Comment:
Please remove ```.

Also add the outputs. Please follow the guide I linked before to test.

Reply:
The outputs has numpy arrays which if i copy and put them it won't look good. I suggest that i should add (```) as the guide says for code that can not be tested for some reasons. Please let me know your opinion thank you.

Reply:
Or I should just allow the code like that?

Reply:
>  I suggest that i should add (```) as the guide says
No, please keep the doctest. 

> copy and put them it won't look good
It will look good.

Also, please use a smaller image size and if possible use tf.constant so that the value won't change on every run.

Reply:
okay

Reply:
a smaller image of about what size?

Reply:
Maybe (3, 3) or (2,2)?

Reply:
I have made the changes and included the outputs